pkg/sql/catalog/lease/storage.go

```diff
@@ -116,8 +119,17 @@ func (s storage) crossValidateDuringRenewal() bool {
 // or offline (currently only applicable to tables), the error will be of type
 // inactiveTableError. The expiration time set for the lease > minExpiration.
 func (s storage) acquire(
-	ctx context.Context, minExpiration hlc.Timestamp, id descpb.ID,
-) (desc catalog.Descriptor, expiration hlc.Timestamp, prefix []byte, _ error) {
+	ctx context.Context,
+	minExpiration hlc.Timestamp,
+	previousVersion descpb.DescriptorVersion,
```

Comment:
Update the comment to explain why `previousVersion` is in the input parameters?

Reply:
+1